src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -96,6 +97,89 @@ class AppStoreConnectArgument(cli.Argument):
     )
 
 
+class AppStoreVersionArgument(cli.Argument):
+    APP_STORE_VERSION = cli.ArgumentProperties(
+        key='app_store_version',
+        flags=('--app-store-version',),
+        description=(
+            'Version of the build published to App Store '
+            'that identifies an iteration of the bundle. '
+            'The string can only contain one to three groups of numeric characters (0-9) '
+            'separated by period in the format [Major].[Minor].[Patch]'
```

Comment:
Maybe add an example?

```suggestion
            'separated by period in the format [Major].[Minor].[Patch]. '
            'For example `3.2.46`'
```